history-of-quakers/modernized/22-chapter-20.adoc

```diff
@@ -622,22 +678,32 @@ Do not think I go from what I spoke to you the last evening;
 I only stay to do it in a way that is answerable before God and man:
 I can say no more now, but recommend to your prayers,
 
+[.signed-section-closing]
 Your true friend,
 
-ELIZABETH.
+[.signed-section-signature]
+Elizabeth.
 
+[.postscript]
 P+++.+++ S. I almost forget to tell you, that my sister writes me word,
 she had been glad you had taken your journey by Osenburgh, to return to Amsterdam.
 There is also a Drossard of Limbourg near this place,
 (to whom I gave an exampler of R. B.`'s apology,)
 very desirous to speak with some of his friends.
 
+--
+
 Yet another letter W. Penn received from the said princess,
 in answer to one he wrote from the Briel, at his passage towards England,
 which was as follows:
 
+[.embedded-content-document.letter]
+--
+
+[.letter-heading]
 To the princess Elizabeth, Salvation in the Cross, Amen.
```

Comment:
Not sure if this should be [.letterhead] or not

Reply:
yeah, it's weird. i think it's fine as a letter head though.